hydromt_sfincs/utils.py

```diff
@@ -317,7 +317,7 @@ def write_timeseries(
     data[:, 0] = (df.index - tref).total_seconds()
     # calculate required width for time column; hard coded single decimal precision
     # format for other columns is based on fmt`argument
-    w = int(np.floor(np.log10(data[-1, 0]))) + 3
+    w = int(np.floor(np.log10(abs(data[-1, 0])))) + 3
```

Comment:
Accidently found this "bug". This line errors if tref > tstart. This shouldn't be the case normally and not sure how SFINCS deals with it. With the abs statement it always works.